src/lib/mpq/chain.js

```diff
@@ -0,0 +1,40 @@
+import path from 'path';
+
+import glob from 'globby';
+
+import MPQ from './';
```

Comment:
Any reason for the double spacing between imports?

Reply:
The `import/order` rule enforces that `path` – a Node module – comes before `globby` – a regular external module. If we were to group these two together, they'd be in the wrong ASCII order. But I agree it looks a bit wonky 😞